src/prune-fhir-schema.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import fhir_schema from '../schema/fhir-schema-full.json';
```

Comment:
It seems that the only place we use the full fhir schema is in this new script, right? Perhaps it'd be best to document where this script comes from, add the capability to fetch it if needed, and remove it from our repo. This would remove a _big_ file.